howler-audio-source.js

```diff
@@ -28,27 +28,30 @@ export class HowlerAudioSource extends Component {
     };
 
     start() {
+        this.lastPlayedAudioId = null;
+        this.origin = new Float32Array(3);
+        this.lastOrigin = new Float32Array(3);
+
         this.audio = new Howl({
             src: [this.src],
             loop: this.loop,
             volume: this.volume,
-            autoplay: this.autoplay,
+            autoplay: false,
+            onload:()=>{
+                if (this.spatial) {
+                    this.object.getPositionWorld(this.origin);
+                }
+                if (this.autoplay) {
+                    this.play();
```

Comment:
Maybe `this.updatePosition()` should be called immediately after play to make sure the sound isn't emitted from an incorrect position before the first update() call?

Reply:
Yes, I was thinking about this. Maybe it's best to start autoplay on the first update after the audio is loaded. I should also convert the file to TypeScript if I'm making edits.